test/util/velero/velero_utils.go

```diff
@@ -96,6 +96,13 @@ var pluginsMatrix = map[string]map[string][]string{
 		"gcp":     {"velero/velero-plugin-for-gcp:v1.8.0"},
 		"csi":     {"velero/velero-plugin-for-csi:v0.6.0"},
 	},
+	"v1.13": {
+		"aws":     {"velero/velero-plugin-for-aws:v1.9.0"},
+		"azure":   {"velero/velero-plugin-for-microsoft-azure:v1.9.0"},
+		"vsphere": {"vsphereveleroplugin/velero-plugin-for-vsphere:v1.5.1"},
```

Comment:
Is vsphere plugin the only one that doesn't bump with new velero version?

Reply:
There is a new tag https://github.com/vmware-tanzu/velero-plugin-for-vsphere/releases/tag/v1.5.2.
Not sure whether we should bump.

Reply:
Let me check with @yanji09.

Reply:
Got the confirmation, we can bump vSphere plugin to 1.5.2 in Velero v1.13, I will modify this PR once several successful nightly tests. Thanks @kaovilai.